src/ehr_components/common_components/autoComplete.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { Fragment, useState, useContext } from "react";
 import { Spinner } from "react-bootstrap";
 import { ClearButton, Typeahead } from "react-bootstrap-typeahead";
-import options from "../options";
 import "react-bootstrap-typeahead/css/Typeahead.css";
 
 function AutoComplete(props) {
+
+  let { options, multiple } = props;
```

Comment:
`options` should be accepted as a prop as it will be different for each `<AutoComplete/>` instance